<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>laser_line_writer</class>
 <widget class="QMainWindow" name="laser_line_writer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>577</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Simple XY Scan</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,0,0" columnstretch="0,0">
    <item row="0" column="0" rowspan="2" colspan="2">
     <widget class="QGroupBox" name="control_groupBox">
      <property name="title">
       <string>controll</string>
      </property>
      <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,0,0,0">
       <item row="1" column="4">
        <widget class="QLabel" name="progres_label">
         <property name="text">
          <string>Progress:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QPushButton" name="start_pushButton">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2" colspan="2">
        <widget class="QPushButton" name="interrupt_pushButton">
         <property name="text">
          <string>Interrupt</string>
         </property>
        </widget>
       </item>
       <item row="1" column="5">
        <widget class="QDoubleSpinBox" name="progress_doubleSpinBox"/>
       </item>
       <item row="0" column="2" colspan="2">
        <widget class="QPushButton" name="add_item_pushButton">
         <property name="text">
          <string>add item</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QComboBox" name="add_item_comboBox"/>
       </item>
       <item row="0" column="5">
        <widget class="QPushButton" name="update_move_display_pushButton">
         <property name="text">
          <string>Update Move Display</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QGroupBox" name="plot_groupBox">
      <property name="title">
       <string>Plot</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2"/>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QGroupBox" name="verticalGroupBox">
      <property name="title">
       <string>items</string>
      </property>
      <layout class="QVBoxLayout" name="items_verticalLayout"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>706</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
